examples/validation-app/README.md

```diff
@@ -19,12 +19,10 @@ instance with the in-memory storage.
    @property({
     type: 'string',
     required: true,
-    // --- add jsonSchema -----
```

Comment:
nitpick: The comment hightlight the new code in the entire file, so better keep them.

Reply:
Re-added with some modification.